lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/LocalRuntimeDriverConfigurationGenerator.java

```diff
@@ -92,7 +92,7 @@ public Configuration writeConfiguration(final File jobFolder,
   }
 
   public static void main(final String[] args) throws InjectionException, IOException {
-    final LocalSubmissionFromCS localSubmission = LocalSubmissionFromCS.fromCommandLine(args);
+    final LocalSubmissionFromCS localSubmission = LocalSubmissionFromCS.fromBootstrapConfigFile(args[0]);
```

Comment:
Change the parameter to be a `File`. Then, check that the file `.exists` and is readable and produce error messages if not. The same applies to the YARN version of the code.